src/tactic/positivity.lean

```diff
@@ -345,10 +345,38 @@ add_tactic_doc
 
 end interactive
 
-variables {α R : Type*}
+variables {ι α R : Type*}
 
 /-! ### `positivity` extensions for particular arithmetic operations -/
 
+section function
+open function
+variables (ι) [has_zero α] {a : α}
+
+private lemma function_const_nonneg_of_pos [preorder α] (ha : 0 < a) : 0 ≤ const ι a :=
+const_nonneg_of_nonneg _ ha.le
```

Comment:
```suggestion
function.const_nonneg_of_nonneg _ ha.le
```

Reply:
Actually, I think `order.pi` is not imported here; so maybe this extension belongs in `order.pi` instead.

Reply:
We discussed a week or so ago on Zulip that we should be careful with adding more imports (especially deep down in the hierarchy), so I'd recommend either adding this extension to `order.pi` or, if `positivity` isn't available there, creating a new file.

Reply:
Added it to `algebra.order.pi` for now.